src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.managed.cs

```diff
@@ -441,6 +462,7 @@ private void UpdateLayout(double? extentAdjustment, bool isScroll)
 
 			UnfillLayout(extentAdjustment ?? 0);
 			FillLayout(extentAdjustment ?? 0);
+			UpdateDynamicSeed(null, null);
```

Comment:
Maybe make the parameters null by default?

Reply:
we are actually setting values of seed to `null` and `null`
i think it would be better to be explicit here

```
UpdateDynamicSeed();
UpdateDynamicSeed(null, null);
```
Have complete different meaning.
But, for clarity, I will change it to SetDynamicSeed.